pkg/resource/api_test.go

```diff
@@ -71,13 +73,13 @@ func TestCreate(t *testing.T) {
 				client: &test.MockClient{
 					MockCreate: test.NewMockCreateFn(errBoom),
 				},
-				typer: MockSchemeWith(&MockClaim{}, &MockClass{}, &MockManaged{}),
+				typer: fake.MockSchemeWith(&fake.MockClaim{}, &fake.MockClass{}, &fake.MockManaged{}),
```

Comment:
`fake.MockFoo` feels a bit stutter-ey.  Now that these are in their own package I'd prefer they were named just `fake.Scheme`, `fake.Claim`, etc.